src/GraphQL.MicrosoftDI.Tests/TestExtensions.cs

```diff
@@ -0,0 +1,14 @@
+using System.Threading.Tasks;
+using Shouldly;
+
+namespace GraphQL.MicrosoftDI.Tests
+{
+    internal static class TestExtensions
+    {
+        public static void ShouldBeTask(this object value, object expected)
+        {
+            value.ShouldBeAssignableTo(typeof(Task<object>));
+            ((Task<object>)value).Result.ShouldBe(expected);
```

Comment:
```suggestion
            value.ShouldBeAssignableTo(typeof(Task<object>)).Result.ShouldBe(expected);
```

Reply:
This code does not compile.  Reverted.

Reply:
My bad. Of course `value.ShouldBeAssignableTo<Task<object>>().Result.ShouldBe(expected);`

Reply:
Nice 👍